server/services/caseloadService.ts

```diff
@@ -381,7 +381,27 @@ export default class CaseloadService {
   }
 
   private isRecall = (offender: ManagedCase): boolean => {
-    return offender.nomisRecord?.recall && offender.nomisRecord?.recall === true
+    const recall = offender.nomisRecord?.recall && offender.nomisRecord.recall === true
+
+    // Get the confirmed release date (if present) or CRD, and use in that order of preference
+    const releaseDate = offender.nomisRecord?.confirmedReleaseDate
+      ? moment(offender.nomisRecord.confirmedReleaseDate, 'YYYY-MM-DD')
+      : moment(offender.nomisRecord.conditionalReleaseDate, 'YYYY-MM-DD')
+
+    // Get the post recall release date (if present) or set to undefined.
+    const postRecallReleaseDate = offender.nomisRecord?.postRecallReleaseDate
+      ? moment(offender.nomisRecord?.postRecallReleaseDate, 'YYYY-MM-DD')
+      : undefined
+
+    // If the post-recall release date is AFTER the release date, then it is a genuine recall.
+    // If the post-recall release date is BEFORE or EQUAL to the release date - we ignore the recall flag.
+    // This catches the situation where multiple sentences exist and the recall applies to a previous sentence.
+    if (recall && releaseDate && postRecallReleaseDate) {
+      return postRecallReleaseDate.isAfter(releaseDate)
+    }
```

Comment:
Still a question about when the PRRD is EQUAL to the CRD. This implementation would bring single-sentence recalls into scope of CVL.

Reply:
It would. Jon is having a think about this and talking to Louise. Trying to figure out if there's any other piece of information we could check to signify that there is only one sentence and that the PRRD applies to the same sentence as the release date
.

Reply:
The rules are written out on this ticket and now implemented here - https://dsdmoj.atlassian.net/browse/CVSL-518